Gemfile

```diff
@@ -1,3 +1,15 @@
 source "https://rubygems.org"
 
 gemspec
+
+group :development do
+  gem 'byebug'
+end
+
+group :test do
+  %w[ rspec rspec-core rspec-expectations rspec-mocks rspec-support ].each do |lib|
+    gem lib, github: "rspec/#{lib}"
+  end
```

Comment:
I'd generally recommend you use the latest RSpec release rather than pulling from HEAD like this, unless there is a specific unreleased feature in RSpec that you need.  RSpec is very stable (our CI build is very good at preventing regressions!) but the latest release is even more stable :).

In Vanguard we pin to the latest RSpec specifically because I like using HEAD on my main projects so that I can run into any inconsistencies, etc from new features before they are released.

Still, if you like being on the edge, this is totally fine :).